### DISCUSSION TITLE
Help to Server Migration

### DISCUSSION THREAD

**Discussion by manishonc** [pocketbase/pocketbase#4119] [Q&A]
Hi am a no coder but knows basic.
I have deployed the pocket base on AWS EC2 .
It was working fine.
Currently my app have 80k users .and I am serving them with pocket base.

But now I want to transfer from EC2 to Digital ocean because of cost.

I tried backup and restore method.

But when I backup the pocketbase. The pb_back file is almost 4GB and every day it's increasing.

But I haven't added any new data . And I am not getting any users generated data also.

Why is size is getting bigger .
Is there something like log o can delete . So become smaller so I can migrate it to new server 
Please help me my bill is getting higher every second .

**Comment by ganigeorgiev** [comment]
You can try temporary disabling the logs from the _Admin UI > Logs > Logs Settings > Max days retention_ and set it to `0`.

Optionally, you can also use the [sqlite3 cli tool](https://sqlite.org/cli.html) run `VACUUM` for the database file:
```js
sqlite3 ./pb_data/data.db 'VACUUM'
```

**Reply by manishonc** [reply]
In the UI I don't see any logs.

But I am going to try VACCUM Method.
I will let you know what happens

Thanks for quick reply

**Reply by santoshgistto** [reply]
<img width="1473" alt="Screenshot 2024-01-10 at 11 41 25 AM" src="https://github.com/pocketbase/pocketbase/assets/12577046/f611ad28-7c2a-4c11-aedc-52764c46127f">

@manishionc FYI

**Reply by ganigeorgiev** [reply]
Either you are not looking at the right place or you are just not reporting properly the size.

> The pb_back file is almost 4GB and every day it's increasing.
> But I haven't added any new data . And I am not getting any users generated data also.

I have no way to confirm that what you are saying is true. Unless you have some custom hooks code, then the only way the backup with the prebuilt executable to grow without new data to be added is the logs to be enabled.

**Reply by manishonc** [reply]
I did the VACCUM . after that I was doing backup it was taking time and after 2 min it automatically logout me and now I am not able to login with my credentials.

**Reply by manishonc** [reply]
![image](https://github.com/pocketbase/pocketbase/assets/84983048/dbff68d8-a4cd-44fb-a7d3-a294848b3f05)

**Reply by manishonc** [reply]
after doing the vaccum - lost login - also now my app is not getting any data now i think i breaked it.
the app is in production - what do i do please help me

**Reply by ganigeorgiev** [reply]
Running `VACUUM` shouldn't "break" your app.

If the app is not starting check your systemd service (if you have such) logs file.

You are definitely doing something wrong but at the same time you are not providing any details to understand what is going on and there is no way for me to assist you fixing the issue so I'll stop engaging further with the discussion.

**Reply by manishonc** [reply]
Can you specify what information do you need ?
So I can provide.
Also the pocketbase is running it's just I am not able to login now.
and the restapi is not returning any data.

But I have the backup file .
If you can guide me how can I use that back-up file to revert back to the way it was it would be helpful.

But if you feel like I am wasting your time..
You don't have to engage with this discussion.

Thankyou very much . 
For the product.

Have a good day.
You can close the discussion.
